data/templates/accel-ppp/ipoe.config.tmpl

```diff
@@ -25,11 +25,21 @@ level=5
 verbose=1
 {% for interface in interfaces %}
 {% if interface.vlan_mon %}
-interface=re:{{ interface.name }}\.\d+,{% else %}interface={{ interface.name }},{% endif %}shared={{ interface.shared }},mode={{ interface.mode }},ifcfg={{ interface.ifcfg }},range={{ interface.range }},start={{ interface.sess_start }},ipv6=1
+interface=re:{{ interface.name }}\.\d+,{% else %}interface={{ interface.name }},{% endif %}shared={{ interface.shared }},mode={{ interface.mode }},ifcfg={{ interface.ifcfg }}{{ ',range=' + interface.range if interface.range is defined and interface.range is not none }},start={{ interface.sess_start }},ipv6=1
```

Comment:
I'm generally against hardline line length restrictions, but I think this line is really way too long. Do you think we can safely break it down into shorter lines?

Reply:
It's actually even worse - the `else` and `endif` block is terminated somewhere within the line, but started outside - this is really confusing.

If you de-nest it you get:

```
{% for interface in interfaces %}
{%   if interface.vlan_mon %}
interface=re:{{ interface.name }}\.\d+,
{%   else %}
interface={{ interface.name }},
{%   endif %}
shared={{ interface.shared }},mode={{ interface.mode }},ifcfg={{ interface.ifcfg }}{{ ',range=' + interface.range if interface.range is defined and interface.range is not none }},start={{ interface.sess_start }},ipv6=1
{% endfor %}
```

I'd write it better human readable as som,ething like:



```
{% for interface in interfaces %}
{%   set ifname = interface.name %}
{%   if interface.vlan_mon %}
{%     set ifname = 're:' ~ interface.name ~ '\.\d+' %}
{%   endif %}
interface={{ ifname }},shared={{ interface.shared }},mode={{ interface.mode }},ifcfg={{ interface.ifcfg }}{{ ',range=' ~ interface.range if interface.range is defined and interface.range is not none }},start={{ interface.sess_start }},ipv6=1
{% endfor %}
```

Reply:
> I'd write it better human readable as som,ething like:

@c-po  Done

> I'm generally against hardline line length restrictions, but I think this line is really way too long. Do you think we can safely break it down into shorter lines?

@dmbaturin each interface can work in different modes l2/l3 ipv6 enabled/disabled vlan_mon used/not used etc.
I don't think it can be reduced
at least I don't see a solution for this yet